index.js

```diff
@@ -2,7 +2,7 @@ var postcss = require('postcss');
 var path = require('path');
 var fs = require('fs');
 
-function getSortOrder(options) {
+function getSortOrderFromOptions(options) {
```

Comment:
Oh, I see you use `FromOptions` suffix for another option. I'll reconsider my opinion about that. Sorry, I've read the code function by function :)